### DISCUSSION TITLE
Blog Plugin: Can I control where the Archive link goes?

### DISCUSSION THREAD

**Discussion by pitosalas** [squidfunk/mkdocs-material#6641] [Q&A] [answered]
Here is my blog so far. (The source is public too): https://pitosalas.github.io/wp-convert/

I would like the blog archive link to appear in the top level nav. Is that possible?

Thanks!

**Comment by markallasread** [comment]
Mind detailing your question a bit further Pito?
Thanks in advance!

**Comment by alexvoss** [comment, marked as answer]
I assume you want to have the years at the top level? Would you be happy if the archive was simply automatically expanded? That might be easier to achieve with [auto-expanding sections](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-expansion).

**Comment by pitosalas** [comment]
Great idea, thanks. I tried that feature and it does the job.

Thanks,

Pito
